router/router.go

```diff
@@ -171,6 +172,14 @@ func Load(middleware ...gin.HandlerFunc) http.Handler {
 		debugger.GET("/pprof/trace", debug.TraceHandler())
 	}
 
+	monitor := e.Group("/api/metrics")
+	{
+		monitor.GET("/prometheus",
+			session.MustAdmin(),
```

Comment:
Since drone is also run as a saas product (at drone.io) the metrics need to be restricted to the system admins.